backend/src/Squidex/Areas/Api/Controllers/Schemas/Models/SchemaDto.cs

```diff
@@ -103,6 +103,11 @@ protected virtual void CreateLinks(Resources resources)
                 AddGetLink("contents", resources.Url<ContentsController>(x => nameof(x.GetContents), values));
             }
 
+            if (resources.CanReadOwnContent(Name))
+            {
+                AddGetLink("contents", resources.Url<ContentsController>(x => nameof(x.GetContents), values));
```

Comment:
The link name "contents" should be unique.

Reply:
If should be unique which name should I write for `resources.CanReadOwnContent(Name)`?

Reply:
The resources is a dictionary, when you add 2 resources with the same name "update" you are overriding the previous value.

We only need to check for own permissions here in the API.